components/ViewMoreLessButton.js

```diff
@@ -4,7 +4,7 @@ import { solid } from '@fortawesome/fontawesome-svg-core/import.macro'
 
 export default function ViewMoreLessButton(props) {
   return (
-    <button
+    <div
```

Comment:
Changing buttons to divs breaks accessibility. Can this be investigated further?

Reply:
Could we potentially use the details & summary component here instead?